style.scss

```diff
@@ -76,8 +84,18 @@ body {
   overflow-x: hidden;
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1 {
```

Comment:
By separating h1 out here, now h1 is bold, because headings are bold by default. But I thought only the word "Sandstorm" in the front-page title was supposed to be bold?